packages/core/src/memory/memory.ts

```diff
@@ -49,7 +49,7 @@ export abstract class MastraMemory extends MastraBase {
       this.vector = config.vector;
     } else {
       this.vector = new DefaultVectorDB({
```

Comment:
can you add a check if the file exists, if not we move to the new dir?

Reply:
Good idea! [`b772288` (#2662)](https://github.com/mastra-ai/mastra/pull/2662/commits/b7722881af50b3785396aded23eefeade218df4c)